package.json

```diff
@@ -28,43 +28,45 @@
     "i18next-localstorage-backend": "^3.1.3",
     "internal-nav-helper": "^3.1.0",
     "intl-messageformat": "^9.12.0",
-    "ipfs-core": "^0.13.0",
     "ipfs-css": "^1.3.0",
-    "ipfs-http-client": "^55.0.0",
-    "ipld": "^0.21.1",
-    "ipld-bitcoin": "^0.1.9",
-    "ipld-dag-cbor": "^0.13.1",
-    "ipld-dag-pb": "^0.15.2",
-    "ipld-ethereum": "^2.0.3",
-    "ipld-explorer-components": "^2.5.1",
-    "ipld-git": "^0.2.3",
-    "ipld-raw": "^2.0.1",
-    "ipld-zcash": "^0.4.1",
+    "ipld-explorer-components": "file:../../ipfs/ipld-explorer-components/ipld-explorer-components-3.0.3.tgz",
```

Comment:
will be updated once https://github.com/ipfs/ipld-explorer-components/pull/360 is merged